data/human/news.txt

```diff
@@ -2190,3 +2380,9 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+			"To the moon, and not coming back."
+			"On a small space-station / early in the morning / seven spaceships landed / nicely in a line / and the captain / transmitted departure / tick tick woosh woosh / here we go"
+			"Time flies like an Arrow, fruit flies like a Banana."
+			"The Panda eats shoots and leaves. The hunter eats, shoots and leaves."
+			"While the man was hunting the deer ran through the forest."
```

Comment:
```suggestion
			"While the man was hunting the deer it ran through the forest."
```
Unless it's intended not to make as much sense.

Reply:
I'll remove it. It doesn't work here.